packages/headless/src/features/facets/facet-search-set/facet-search-reducer-helpers.ts

```diff
@@ -2,11 +2,18 @@ import {FacetSearchOptions} from './facet-search-request-options';
 import {FacetSearchResponse} from '../../../api/search/facet-search/facet-search-response';
 import {FacetSearchRequestOptions} from '../../../api/search/facet-search/base/base-facet-search-request';
 
+export interface FacetSearchStateOptions extends FacetSearchRequestOptions {
```

Comment:
Rather than extending the `FacetSearchRequestOptions` interface, wouldn't it have been simpler to add `initialNumberOfValues` as a property of `FacetSearchState`?

I think it would be cleaner to not group this property with the `FacetSearchRequestOptions` interface, because it is not part of the request. It's just for managing client-side state. e.g.

```
export type FacetSearchState<T extends FacetSearchResponse> = {
  options: FacetSearchRequestOptions;
  isLoading: boolean;
  response: T;
  initialNumberOfValues: number;
};
```

Reply:
I found it more contextually fitting to put it in options, outside felt out of place, I must have been confused by the name